resolwe/flow/tests/test_commands.py

```diff
@@ -139,7 +139,7 @@ def setUp(self):
         super().setUp()
 
         # Make sure the test processes are in the database
-        self._register_schemas(path=[PROCESSES_DIR])
+        self._register_schemas(processes_path=[PROCESSES_DIR])
```

Comment:
I would write `_paths` in plural since the input is a list of paths. Thus I suggest `process_paths` or `proc_paths`.